docx/helper/helper.go

```diff
@@ -17,6 +17,7 @@ func ParseXML(content []byte) (xmlDoc *xml.XmlDocument, err error) {
 	// http://stackoverflow.com/a/27475227/358804
 	xp := xmlDoc.DocXPathCtx()
 	xp.RegisterNamespace("w", "http://schemas.openxmlformats.org/wordprocessingml/2006/main")
+	xp.RegisterNamespace("w14", "http://schemas.microsoft.com/office/word/2010/wordml")
```

Comment:
@isimluk receiving error on this new URL:

``The resource you are looking for has been removed, had its name changed, or is temporarily unavailable.``

Reply:
Looked the URL up on the microsoft website - it's fine

ref: https://docs.microsoft.com/en-us/openspecs/office_standards/ms-docx/666d587b-fa8d-4515-96e9-b2e300cd3d77

Reply:
Yeah, in theory there should by documentation or XSD on the xmlns uri. BUT in real world, the xmlns uris rarely keep working. Fortunatelly?, XML standard does not require xmlns uri to be working...